third_party/txt/src/txt/font_collection.cc

```diff
@@ -225,6 +229,8 @@ std::shared_ptr<minikin::FontFamily> FontCollection::CreateMinikinFontFamily(
     minikin_fonts.emplace_back(std::move(minikin_font));
   }
 
+  TRACE_EVENT1("flutter", "minikin::FontFamily",
```

Comment:
I think a trace counter makes more sense here.

Reply:
I tried that in my first version and it's a little confusing to me because it no longer appears in the UI thread. Instead, if has its own grey box on the very top (similar to CPU/GPU clock counters). Is there a way to trace a counter, but show that as a normal box in the UI thread?